packages/graphics/kmscube/package.mk

```diff
@@ -0,0 +1,31 @@
+################################################################################
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2016 - present Team LibreELEC
```

Comment:
`Copyright (C) 2016-present Team LibreELEC`

Reply:
If the file was first created in 2018 it will be "Copyright (C) 2018 Team LibreELEC"